src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogGraphListener.java

```diff
@@ -87,14 +92,16 @@ public void onNewHead(FlowNode flowNode) {
             return;
         }
         try {
+            String result = getResultTag(endNode);
             BuildData buildData = new BuildData(run, flowNode.getExecution().getOwner().getListener());
             String hostname = buildData.getHostname("");
             Map<String, Set<String>> tags = buildData.getTags();
             TagsUtil.addTagToTags(tags, "stage_name", getStageName(startNode));
             TagsUtil.addTagToTags(tags, "parent_stage_name", directParentName);
             TagsUtil.addTagToTags(tags, "stage_depth", String.valueOf(stageDepth));
             tags.remove("result"); // Jenkins sometimes consider the build has a result even though it's still running.
```

Comment:
Is this still needed if the value gets overwritten in the next line?

Reply:
True, I can probably remove this line!